scripts/diagnostics-agent-setup.ps1

```diff
@@ -0,0 +1,84 @@
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$DiagnosticsWindowsBinariesURL
+)
+
+$ErrorActionPreference = "Stop"
+
+$utils = (Resolve-Path "$PSScriptRoot\Modules\Utils").Path
+Import-Module $utils
+
+$variables = (Resolve-Path "$PSScriptRoot\variables.ps1").Path
+. $variables
+
+function New-DiagnosticsEnvironment {
+    $service = Get-Service $DIAGNOSTICS_SERVICE_NAME -ErrorAction SilentlyContinue
+    if($service) {
+        Stop-Service -Force -Name $DIAGNOSTICS_SERVICE_NAME
+        & sc.exe delete $DIAGNOSTICS_SERVICE_NAME
+        if($LASTEXITCODE) {
+            Throw "Failed to delete exiting $DIAGNOSTICS_SERVICE_NAME service"
+        }
+        Write-Output "Deleted existing $DIAGNOSTICS_SERVICE_NAME service"
+    }
+    New-Directory -RemoveExisting $DIAGNOSTICS_DIR
+    New-Directory $DIAGNOSTICS_BIN_DIR
+    New-Directory $DIAGNOSTICS_LOG_DIR
+    New-Directory $DIAGNOSTICS_SERVICE_DIR
+    New-Directory $DIAGNOSTICS_CONFIG_DIR
+}
+
+function Add-ToSystemPath {
```

Comment:
This is a general purpose function. I should have added it to the `Utils` module as well.
You already import the `Utils` module, so you can consume it from there if it's present.

Can you please:
- move this function at the end of `Utils.psm1`: https://github.com/soccerGB/dcos-windows/blob/master/scripts/Modules/Utils/Utils
- add this function to the list of exported functions over here: https://github.com/soccerGB/dcos-windows/blob/master/scripts/Modules/Utils/Utils.psd1#L46-L52

After you do these changes, you can use `Add-ToSystemPath` from the `Utils` module.

Reply:
I will refactor it and make the necessary changes